apps/notifications/sendNotifications.js

```diff
@@ -0,0 +1,43 @@
+const execSync = require('child_process').execSync;
+const { JsonFile, ConsoleTerminalProvider, Terminal } = require('@rushstack/node-core-library');
+
+// the branch will be changed to main once pr is in
+const notificationResponse = execSync(
```

Comment:
Since we're already using `@rushstack/node-core-library`, makes sense to use `Executable` from that package instead of `execSync` directly.